src/index.js

```diff
@@ -35,10 +35,10 @@ let edgesWaitingInstances = getEdgesWaitingInstances();
 // Add Overlay on activities
 activitiesMonitoringData.forEach((value, key) => {
   // running on time
-  if (value.onTime != false) {
+  if (value.onTime) {
     bpmnVisualization.bpmnElementsRegistry.addOverlays(key, {
       position: "top-center",
-      label: value.onTime,
+      label: String(value.onTime),
```

Comment:
If you prefer, we could use instead a [string template](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)
```suggestion
      label: `${value.onTime}`,
```

Reply:
I prefer the suggestion of @tbouffard.

Reply:
Well, the survey is
- @assynour `String`
- @csouchet string template
- @tbouffard I don't care!

😵‍💫

Reply:
decision taken collectively: use string template